api/generated/write.json

```diff
@@ -74,10 +74,24 @@
                     "type": "string",
                     "description": "The mode of write operation. The only supported mode currently is synchronous.",
                     "enum": [
-                      "synchronous"
+                      "synchronous",
+                      "asynchronous"
```

Comment:
If mode is async, then the API response will only have operation ID. You need to remove "result" from the `required` list in the API reponse.